### DISCUSSION TITLE
Download Video Option?

### DISCUSSION THREAD

**Discussion by OmniiOMEGA** [YTLitePlus/YTLitePlus#139] [Ideas]
Is there a Download Video feature?

**Comment by Balackburn** [comment]
> Is there a Download Video feature?

No, I recommend you to use this shortcut if you want to download videos : 

https://routinehub.co/shortcut/17265/

Click share and use the shortcut you can rename it into YouTube Downloader if you want

**Reply by OmniiOMEGA** [reply]
> > Is there a Download Video feature?
> 
> No, I recommend you to use this shortcut if you want to download videos : 
> 
> https://routinehub.co/shortcut/17265/
> 
> Click share and use the shortcut you can rename it into YouTube Downloader if you want

Where does it download to on iPhone? 
It also didn't let me pick quality ?

**Reply by Balackburn** [reply]
> > > Is there a Download Video feature?
> > 
> > No, I recommend you to use this shortcut if you want to download videos : 
> > 
> > https://routinehub.co/shortcut/17265/
> > 
> > Click share and use the shortcut you can rename it into YouTube Downloader if you want
> 
> Where does it download to on iPhone? 
> It also didn't let me pick quality ?

Your photo library, you can pick quality by clicking advanced instead of auto.

Also you can edit the shortcut to download it in files instead.

**Reply by OmniiOMEGA** [reply]
> > > > Is there a Download Video feature?
> > > 
> > > No, I recommend you to use this shortcut if you want to download videos : 
> > > 
> > > https://routinehub.co/shortcut/17265/
> > > 
> > > Click share and use the shortcut you can rename it into YouTube Downloader if you want
> > 
> > Where does it download to on iPhone? 
> > It also didn't let me pick quality ?
> 
> Your photo library, you can pick quality by clicking advanced instead of auto.

That's what I thought it would do, but nothing is appearing in my photo library?

**Reply by Balackburn** [reply]
> > > > > Is there a Download Video feature?
> > > > 
> > > > No, I recommend you to use this shortcut if you want to download videos : 
> > > > 
> > > > https://routinehub.co/shortcut/17265/
> > > > 
> > > > Click share and use the shortcut you can rename it into YouTube Downloader if you want
> > > 
> > > Where does it download to on iPhone? 
> > > It also didn't let me pick quality ?
> > 
> > Your photo library, you can pick quality by clicking advanced instead of auto.
> 
> That's what I thought it would do, but nothing is appearing in my photo library?

Idk it works normally try again otherwise edit the shortcut to save in files instead
